src/views/payment-sheet-views/card-view.js

```diff
@@ -443,12 +445,20 @@ CardView.prototype._isCardTypeSupported = function (cardType) {
   return supportedCardTypes.indexOf(configurationCardType) !== -1;
 };
 
-function camelCaseToSnakeCase(string) {
+function kebabCaseToCamelCase(string) {
   return string.replace(/([a-z])([A-Z])/g, '$1-$2').toLowerCase();
 }
 
 function capitalize(string) {
   return string[0].toUpperCase() + string.substr(1);
 }
 
+function normalizeStyles(styles) {
+  Object.keys(styles).forEach(function (style) {
+    var newKey = kebabCaseToCamelCase(style);
+
+    styles[newKey] = styles[style];
```

Comment:
Think I'd be fine one-lining this, even though I was the one who split it apart in the first place.